addon/utils/options-and-events.js

```diff
@@ -204,7 +204,11 @@ export function addEventListener(
   let isOnce = isOnceEvent(originalEventName);
 
   let nativeListener = (target, eventName, callback) => {
-    target.addEventListener(eventName, callback, { once: isOnce });
+    if (target?.nodeName === 'GMP-INTERNAL-AM') {
```

Comment:
Unsure if there's a better way of doing this now but this works like this. ☝️

Reply:
Does `addListener` support options? I don't see it in the docs.
I see that the `addListenerOnce` is still there, which we use a few lines later.

If it doesn't and the advanced markers pass the `isDom` check, then I guess we could integrate the `nodeName` check into the `isDom` check.

Reply:
Looks like it supports the `addListenerOnce`.
I'll do this 👇 
> If it doesn't and the advanced markers pass the isDom check, then I guess we could integrate the nodeName check into the isDom check.